code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -250,13 +233,12 @@
 
 	update_visuals()
 
-	var/remove = TRUE
-	if(our_air.temperature > MINIMUM_TEMPERATURE_START_SUPERCONDUCTION)
-		if(consider_superconductivity(starting = 1))
-			remove = FALSE
-
-	if ((!our_excited_group && remove) || (cached_atmos_cooldown > (EXCITED_GROUP_DISMANTLE_CYCLES * 2)))
+	#define REMOVE (!(our_air.temperature > MINIMUM_TEMPERATURE_START_SUPERCONDUCTION && consider_superconductivity(starting = TRUE)))
+	//am i really so obsessed with performance to use a temporary define instead of a var here?
```

Comment:
why not just write it out instead of writing it out and having it be copy-pasted in by the preprocessor

Reply:
Agreed